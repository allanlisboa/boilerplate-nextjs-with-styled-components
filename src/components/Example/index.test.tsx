import { render, screen } from '@testing-library/react'

import Example from './index'

describe('<Example />', () => {
  it('shold render the heading', () => {
    const { container } = render(<Example />)

    expect(
      screen.getByRole('heading', { name: /Boilerplae NextJS/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
