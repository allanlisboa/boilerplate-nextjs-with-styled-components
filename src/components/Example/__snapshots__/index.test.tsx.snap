// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> shold render the heading 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 3rem;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  text-align: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  font-size: 2.5rem;
}

.c2 {
  font-size: 2rem;
  font-weight: 300;
  margin-top: 1rem;
}

<div
  class="c0"
>
  <h1
    class="c1"
  >
    Boilerplae NextJS
  </h1>
  <h2
    class="c2"
  >
    Typescript, ReactJS, NextJS e Styled Components
  </h2>
</div>
`;
